crates/miden-tx/src/host/mod.rs

```diff
@@ -66,6 +68,30 @@ pub use tx_progress::TransactionProgress;
 use crate::auth::SigningInputs;
 use crate::errors::{TransactionHostError, TransactionKernelError};
 
+#[allow(dead_code)]
+pub trait ProcessStateExtension {
+    fn get_stack_word_be(&self, start_idx: usize) -> Word;
+    fn get_stack_word_le(&self, start_idx: usize) -> Word;
+}
```

Comment:
Not sure if we should add `#[allow(dead_code)]` here. This is for the `le` variant, right? Should we remove that method if we don't need it instead? It should be very easy to bring it back if it's needed.

Reply:
https://github.com/0xMiden/miden-vm/issues/2235